src/java/org/apache/cassandra/config/Config.java

```diff
@@ -397,6 +397,8 @@ public static class SSTableConfig
     @Replaces(oldName = "max_mutation_size_in_kb", converter = Converters.KIBIBYTES_DATASTORAGE, deprecated = true)
     public DataStorageSpec.IntKibibytesBound max_mutation_size;
 
+    public volatile boolean bulk_load_enabled = true;
```

Comment:
There is second down below in this file where all guardrail configuration fields are specified. Could you move it down there so they are all logically coupled together?